### PR TITLE
Add editorconfig config

### PR DESCRIPTION
**Pull request by IniZio**
Noticed quite some files e.g. `resources/authgear/templates/en/web/authflowv2/setup_oob_otp.html`, `resources/authgear/templates/en/web/authflowv2/login.html` got tabs accidentally added.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
```
Comment (louischan-oursky) [inline .editorconfig:6]: Not familiar with editorconfig. Does this line mean applying to all files in this repository?

Comment (IniZio) [reply .editorconfig:6]: Yes, see if will want to be more conservative and only cover authui v2 code first?

**Comment by louischan-oursky** [comment]
If the intention is to align the format of those html and css files, then I suggest we set up prettier instead. But prettier does not know how to format Go template :(

**Comment by IniZio** [comment]
Yes I am adding prettier and other lint stuff in https://github.com/authgear/authgear-server/issues/3666. Don't think it will help with the gotcha I mentioned in this pr though :/

**Comment by louischan-oursky** [comment]
Let's put our effort in getting auth flow v2 done first. Formatting is easy when we have a right tool.

**Comment by Ngkaokis** [comment]
Found this https://github.com/NiklasPor/prettier-plugin-go-template

Does it help?

**Comment by IniZio** [comment]
Will close this pr first.

**Pull request closed without merging**